### ISSUE TITLE
train a TSN model with the ResNet50

### ISSUE THREAD

**Issue by xuzero** [yjxiong/temporal-segment-networks#20]
Thanks for your share!
 I want to train a TSN model with the ResNet50, but the log show that:

I0922 21:43:26.134378 35831 solver.cpp:75] Creating training net from net file: home/lk/ResNet/resnet_50/Train_val_resnet_50_ucf101.prototxt
[libprotobuf ERROR google/protobuf/text_format.cc:245] Error parsing text-format caffe.NetParameter: 68:26: Message type "caffe.LayerParameter" has no field named "batch_norm_param".

Then i change the "batch_norm_param" into "BN", then the log shows that:

I0922 22:27:14.272898 11935 caffe.cpp:86] Finetuning from /home/lk/ResNet/resnet_50/ResNet-50-model.caffemodel
F0922 22:27:16.068253 11935 net.cpp:834] Check failed: target_blobs.size() == source_layer.blobs_size() (4 vs. 3) Incompatible number of blobs for layer bn_conv1

It seems the official caffe parameter "batch_norm_param" is different from your own parameter "BN".Would you like to tell me how to fix it?

**Comment by yjxiong** [comment]
The bundled Caffe don't have the `BatchNorm` layer in the official version of Caffe. You can add it to the codebase if necessary. `BN` is not the same thing as `BatchNorm` (more like the combination of `BatchNorm` and `Scale`, as described in the original batch normalization paper).
